source/portable/NetworkInterface/MPS2_AN385/NetworkInterface.c

```diff
@@ -82,6 +82,22 @@ static void prvSetMACAddress( void );
 
 /*-----------------------------------------------------------*/
 
+/*
+ * A pointer to the network interface is needed later when receiving packets.
+ */
+static NetworkInterface_t * pxMyInterface;
+
+static BaseType_t xNetworkInterfaceInitialise( NetworkInterface_t * pxInterface );
```

Comment:
Would you be so kind to make these static function names unique in de source-tree by giving them some sort of a prefix?
Like for instance `xMPS2_NetworkInterfaceInitialise()`?
When every driver uses the same set of function names, a grep "xNetworkInterfaceInitialise" will yield more than 20 results.
And also, a name like `xMPS2_GetPhyLinkStatus()` shows immediately to which driver it belongs.

Reply:
Done, thanks